### PR TITLE
Update note about using atomic blocks with firewall CSP

### PR DESCRIPTION
**Pull request by CatalinFetoiu**
Update note about using atomic blocks with firewall CSP

### CODE REVIEW

File: windows/client-management/mdm/firewall-csp.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ The Firewall configuration service provider (CSP) allows the mobile device manag
 
 > [!NOTE]
 > Firewall rules in the FirewallRules section must be wrapped in an Atomic block in SyncML, either individually or collectively.
+> Atomic blocks are "all or nothing", if a firewall rule or firewall setting in an Atomic block fails to be applied, the entire Atomic block will fail to be applied.
+> If an Atomic block contains a firewall rule or firewall setting that is not supported on a particular Windows OS version, the entire Atomic block will fail to be applied on that Windows version. For example, firewall rules with IcmpTypesAndCodes are only supported on Windows 11, applying an Atomic block that contains a rule with IcmpTypesAndCodes on Windows 10 will fail.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/firewall-csp.md:26]:
```suggestion
>
> Atomic blocks are "all or nothing", if a firewall rule or firewall setting in an Atomic block fails to be applied, the entire Atomic block will fail to be applied.
>
> If an Atomic block contains a firewall rule or firewall setting that is not supported on a particular Windows OS version, the entire Atomic block will fail to be applied on that Windows version. For example, firewall rules with IcmpTypesAndCodes are only supported on Windows 11, applying an Atomic block that contains a rule with IcmpTypesAndCodes on Windows 10 will fail.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @CatalinFetoiu, kindly commit the suggested change. Many thanks.

File: windows/client-management/mdm/firewall-csp.md [hunk 2]
```diff
@@ -22,6 +22,10 @@ The Firewall configuration service provider (CSP) allows the mobile device manag
 
 > [!NOTE]
 > Firewall rules in the FirewallRules section must be wrapped in an Atomic block in SyncML, either individually or collectively.
+>
+> Atomic blocks are "all or nothing", if a firewall rule or firewall setting in an Atomic block fails to be applied, the entire Atomic block will fail to be applied.
```
Comment (vinaypamnani-msft) [inline windows/client-management/mdm/firewall-csp.md:26]:
```suggestion
> Atomic blocks are "all or nothing", if a firewall rule or firewall setting in an Atomic block fails to be applied, the entire Atomic block fails to be applied.
```

File: windows/client-management/mdm/firewall-csp.md [hunk 3]
```diff
@@ -22,6 +22,10 @@ The Firewall configuration service provider (CSP) allows the mobile device manag
...
+>
+> If an Atomic block contains a firewall rule or firewall setting that is not supported on a particular Windows OS version, the entire Atomic block will fail to be applied on that Windows version. For example, firewall rules with IcmpTypesAndCodes are only supported on Windows 11, applying an Atomic block that contains a rule with IcmpTypesAndCodes on Windows 10 will fail.
```
Comment (vinaypamnani-msft) [inline windows/client-management/mdm/firewall-csp.md:28]:
```suggestion
> If an Atomic block contains a firewall rule or firewall setting that is not supported on a particular Windows OS version, the entire Atomic block fails to be applied on that Windows version. For example, firewall rules with IcmpTypesAndCodes are only supported on Windows 11, applying an Atomic block that contains a rule with IcmpTypesAndCodes on Windows 10 fails.
```

**Review by vinaypamnani-msft — APPROVED** [review]

**Pull request merged**